esp-hal/src/clock/clocks_ll/esp32c6.rs

```diff
@@ -211,16 +133,21 @@ pub(crate) fn esp32c6_rtc_bbpll_configure_raw(_xtal_freq: u32, pll_freq: u32) {
         // WAIT CALIBRATION DONE
         while (i2c_mst_ana_conf0_reg_ptr.read_volatile() & I2C_MST_BBPLL_CAL_DONE) == 0 {}
 
+        // workaround bbpll calibration might stop early
+        crate::rom::ets_delay_us(10);
+
         // BBPLL CALIBRATION STOP
         i2c_mst_ana_conf0_reg_ptr.write_volatile(
```

Comment:
in ESP-IDF the order of operations is like this - doesn't seem to make a difference but maybe better to align it